tutors.js

```diff
@@ -0,0 +1,93 @@
+    image: './images/tutors/selected/Jonas.jpg',
+  },
+  {
+    id: '5',
+    name: 'Andrew Mead',
+    position: 'A Full-stack Developer & Teacher',
+    description:
+      "I launched my first Udemy course in 2014 and had a blast teaching and helping others. Since then, I've launched 3 courses with over 110,000 students and over 18,000 5-star reviews.",
+    image: './images/tutors/selected/Andrew.jpg',
+  },
+  {
+    id: '6',
+    name: 'Bucky Roberts',
+    position: 'Director of Engineering',
+    description:
+      'I’ve acquired a wide depth of knowledge and expertise in using my technical skills in programming, computer science, software development, and mobile app development to developing solutions to help organizations increase productivity, and accelerate business performance.',
+    image: './images/tutors/selected/Bucky.jpg',
+  },
+];
+
+const colt = tutors.find((tutor) => tutor.id === '1');
+const maxi = tutors.find((tutor) => tutor.id === '2');
+const brad = tutors.find((tutor) => tutor.id === '3');
+const jonas = tutors.find((tutor) => tutor.id === '4');
+const andrew = tutors.find((tutor) => tutor.id === '5');
+const bucky = tutors.find((tutor) => tutor.id === '6');
+
+// Colt Steel
+document.getElementById('speaker-colt').textContent = colt.name;
+document.getElementById('colt-position').textContent = colt.position;
+document.getElementById('colt-description').textContent = colt.description;
+document.getElementById('colt-image').src = colt.image;
+
+// Maximillian
+document.getElementById('speaker-max').textContent = maxi.name;
+document.getElementById('max-position').textContent = maxi.position;
+document.getElementById('max-description').textContent = maxi.description;
+document.getElementById('max-image').src = maxi.image;
+
+// Brad
+document.getElementById('speaker-brad').textContent = brad.name;
+document.getElementById('brad-position').textContent = brad.position;
+document.getElementById('brad-description').textContent = brad.description;
+document.getElementById('brad-image').src = brad.image;
+
+// Jonas
+document.getElementById('speaker-jonas').textContent = jonas.name;
+document.getElementById('jonas-position').textContent = jonas.position;
+document.getElementById('jonas-description').textContent = jonas.description;
+document.getElementById('jonas-image').src = jonas.image;
+
+// Andrew
+document.getElementById('speaker-andrew').textContent = andrew.name;
+document.getElementById('andrew-position').textContent = andrew.position;
+document.getElementById('andrew-description').textContent = andrew.description;
+document.getElementById('andrew-image').src = andrew.image;
+
+// Bucky
+document.getElementById('speaker-bucky').textContent = bucky.name;
+document.getElementById('bucky-position').textContent = bucky.position;
+document.getElementById('bucky-description').textContent = bucky.description;
+document.getElementById('bucky-image').src = bucky.image;
```

Comment:
**Changes Required**
- [ ] Good effort on storing the tutors as `Array of Objects`. However, I think we can do better, there is a lot of repetition on finding each tutor. Since the design of all the tutor cards are the same, I suggest we use `Array.map` method, iterate the tutors array and assign each tutor property to a **className** dynamically generated.
This will greatly reduce the amount of boilerplate code used.

Reply:
I've loop now on the array and it totally makes my code shorter and no more repetition of my code.
That was a good review. All is working great now. Thanks

Reply:
Well solved